tests/components/geonetnz_quakes/test_geo_location.py

```diff
@@ -182,7 +180,12 @@
         assert len(all_states) == 2
 
         # Test conversion of 200 miles to kilometers.
-        assert mock_feed_init.call_args[1].get("filter_radius") == 321.8688
+        feeds = hass.data[DOMAIN][FEED]
+        assert feeds is not None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5122969)